### ISSUE TITLE
Admin: Shift templates and Schedule templates create HTTP 500

### ISSUE THREAD

**Issue by pitpalme** [coders4help/volunteer_planner#541]
**Describe the bug**
In Admin trying to open Shift templates or Schedule templates creates HTTP 500 response.

**To Reproduce**
1. /admin/
2. Open 'Shift templates' or 'Schedule templates' in app / section 'SCHEDULETEMPLATES'
3. See 'HTTP 500'

**Expected behavior**
I expect, being able to manage shift and schedule templates.

**Additional context**
Error is in "dev", currently deployed is commit `2dfa64b`.

```
Internal Server Error: /admin/scheduletemplates/shifttemplate/

FieldError at /admin/scheduletemplates/shifttemplate/
Cannot resolve keyword 'priority' into field. Choices are: days, ending_time, id, members_only, schedule_template, schedule_template_id, slots, starting_time, task, task_id, workplace, workplace_id
```

Output of `python3 manage.py check` is `System check identified no issues (0 silenced).`.

**Comment by pitpalme** [comment]
Output of `python3 manage.py migrate --check`:
```
Operations to perform:
  Apply all migrations: accounts, admin, auth, blueprint, content, contenttypes, flatpages, news, notifications, organizations, places, registration, scheduler, scheduletemplates, sessions, shiftmailer, sites
Running migrations:
  No migrations to apply.
```

**Comment by pitpalme** [comment]
Merged pending fixes into release/4.0.0.
Current deployed commit is now `7766f5b`.
Error still effective.
